katdal/test/test_chunkstore_s3.py

```diff
@@ -301,14 +301,14 @@ def test_public_read(self):
         y = reader.get_chunk('public/x', slices, x.dtype)
         np.testing.assert_array_equal(x, y)
 
-    @timed(0.4 + 0.2)
+    @timed(0.1 + 0.2)
```

Comment:
The original bug was that you were making timeouts unnecessarily tight for no good reason, and now you're doing it again here...

Reply:
But this is the timeout for the timeout... 😛I want to check that the correct timeout is being used.

Reply:
Hmm, I see. There is no way to guarantee that your process won't get scheduled out for hundreds of milliseconds - a better test would be to using mocking to check what timeout is set on the socket. But we need to get this fixed ASAP, so carry on.